docs/integrate/testing.md

```diff
@@ -0,0 +1,56 @@
+(testing)=
+# Software Testing with CrateDB
+
+This documentation section lists Java- and Python-based test frameworks
```

Comment:
I think it's simpler to just say:
```suggestion
Java and Python based test frameworks...
```

Reply:
Thanks. I amended the patch, using your suggestions.